config.go

```diff
@@ -0,0 +1,52 @@
+package metrics
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"time"
+)
+
+// PushConfig is config for pushing registered metrics to the given URL with the given Interval.
+//
+// URL and Interval are required fields
+type PushConfig struct {
+	// URL defines URL where metrics would be pushed.
+	URL string
```

Comment:
It would be better from usability perspective to remove the required `URL` and `Interval` options from the `PushConfig`. These options must be passed explicitly into `InitPushWithConfig()`. This prevents from passing invalid `PushConfig` to `InitPushWithConfig()` without the required `URL` and `Interval` options.

It would be better also renaming the `PushConfig` to `PushOptions` in order to reflect that all the options inside `PushOptions` are **optional**.